### PR TITLE
tests: add more screenshot tests

### PR DESCRIPTION
**Pull request by jrandolf-2**
Adds a screenshot fixture and adds more test.

### CODE REVIEW

**Comment by OrKoN** [comment]
Looks like tests fail

File: src/bidiMapper/domains/context/BrowsingContextImpl.ts [hunk 1]
```diff
@@ -771,12 +771,12 @@ export class BrowsingContextImpl {
     switch (params.origin) {
       case 'document': {
         script = String(() => {
-          const element = document.documentElement;
+          const rect = document.body.getBoundingClientRect();
```
Comment (OrKoN) [inline src/bidiMapper/domains/context/BrowsingContextImpl.ts:774]: Would not it make the problem worse meaning given that the body can be made of size 0x0 while the content is still visible?

**Review by OrKoN — CHANGES_REQUESTED** [review]

File: src/bidiMapper/domains/context/BrowsingContextImpl.ts (hunk 1 above)
Comment (jrandolf) [reply src/bidiMapper/domains/context/BrowsingContextImpl.ts:774]: Yea, that's a good argument.

**Comment by OrKoN** [comment]
@sadym-chromium assert_screenshot looks useful, let's revert the mapper changes though. WDYT?

**Comment by sadym-chromium** [comment]
> @sadym-chromium assert_screenshot looks useful, let's revert the mapper changes though. WDYT?

agree

**Pull request closed without merging**